ServiceWebsite/ServiceWebsite.AcceptanceTests/Steps/QuestionnaireJourney.cs

```diff
@@ -45,11 +45,17 @@ private void WhenIndividualAttemptsToProceedWithoutProvidingAdditionalInformatio
 
         [When(@"provides additional information '(.*)'")]
         [When(@"provides additional information containing a two character length '(.*)'")]
-        private void WhenIndividualProvidesAdditionalInformationContainingLessThanCharacters(string detail)
+        private void WhenIndividualProvidesAdditionalInformationContainingLessThanThreeCharacters(string detail)
```

Comment:
The method name doesn't match up to the `When` conditions above. Isn't this just `WhenAddingAdditionalInformation`?